_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -0,0 +1,191 @@
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.1-beta.rc4][lnd 0.11.1-beta] is the release candidate for a
+  minor version.  Its release notes summarize the changes as "a number
+  of reliability improvements, some macaroon [authentication token]
+  upgrades, and a change to make our version of [anchor commitments][topic
+  anchor outputs] spec compliant."
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #18267][] and [#19993][Bitcoin Core #19993] add support
+  for [signet][topic signet].  If Bitcoin Core is started with `-signet`
+  (or `-chain=signet`), it'll connect to either the default signet or
+  the signet defined by the `-signetchallenge` and `-signetseednode`
+  parameters.
+
+- [Bitcoin Core #19572][] adds a new "sequence" [ZMQ][bitcoin ZMQ] topic that
+  notifies subscribers when blocks are connected/disconnected and when
+  transactions are added/removed from the mempool. Transaction addition/removal
+  notification messages include a "mempool sequence number" that subscribers can
+  use to determine the order in which these additions/removals occurred and to
+  cross-reference with `getrawmempool` RPC results which now also include this
+  new field.
+
+- [C-Lightning #4068][] and [#4078][C-Lightning #4078] update
+  C-Lightning's signet implementation to be compatible with the final
+  parameters chosen in [BIP325][] and Bitcoin Core's default signet.
+
+- [Eclair #1501][] adds support for unilateral closes of channels using
+  [anchor outputs][topic anchor outputs], completing Eclair's basic
+  implementation of this new LN protocol feature.  According to the PR,
+  Eclair still needs to add the features necessary to fee bump
+  commitments transactions and HTLCs, but this is coming next.
+
+- [LND #4576][] is the first in a planned series of PRs adding support
+  for anchor outputs to LND's [watchtower][topic watchtowers] implementation.  This PR, in
+  particular, adds a flag indicating that anchor outputs were used in
+  the channel being closed so that the watchtower can respond
+  appropriately.  The PR notes, "[the] changes require no modification
+  of the encrypted payload format.  The anchor payloads are equal size
+  and contain exactly the same witness info as the legacy payloads, only
+  requiring light modifications to the reconstruction logic."
+
+- [BIPs #907][] updates the [BIP155][] specification of [version 2 `addr`
+  messages][topic addr v2] to allow addresses up to 512 bytes and adds a new
```

Comment:
The `addrv2` optech topic may need an update.

Reply:
Thanks Jon. Do you mind opening an issue or PR?